app/backend/wells/management/commands/export.py

```diff
@@ -143,11 +143,12 @@ def generate_files(self, zip_filename, spreadsheet_filename):
         #######
         # WELL
         #######
-        well_sql = ("""select well_tag_number, identification_plate_number,
+        well_sql = ("""select well.well_tag_number, identification_plate_number,
```

Comment:
What was the reason behind specifying the table for this column in here? (Just curious)
This SQL could also be better if we left out ambiguous columns and specified the tables for all of them, even by using an alias `well w`...then `w.well_tag_number, w. identification_plate_number`. No need to change it all now, but once your SQL gets big enough to join multiple tables, it's more prone to bugs and invalid data.

Reply:
the newly joined table `well_licences` also has a column `well_tag_number`, so I had to specify which table to select well_tag_number from.